helper_test.go

```diff
@@ -108,6 +125,35 @@ func TestMGetCache(t *testing.T) {
 			}
 		})
 	})
+	// TODO: @SoulPancake I have some doubts about this @rueian
```

Comment:
I have some doubts here, Can you suggest how we can cover the sentinelClient scenario? @rueian

Reply:
Here is an example:  https://github.com/redis/rueidis/blob/63951b5b27a90738fe251a48305f65954df3fec9/sentinel_test.go#L664